include/circt/Dialect/Moore/MooreOps.td

```diff
@@ -650,6 +651,26 @@ def ToBuiltinBoolOp : MooreOp<"to_builtin_bool", [Pure]> {
   }];
 }
 
+def TimeToIntOp : MooreOp<"time_to_int", [Pure]> {
```

Comment:
nit: maye `time_to_logic` is better name? I thought it must produce a two state value because of `_int` suffix.

Reply:
Why is it four valued here? (e.g., I'd have assumed time would be two state valued)

Reply:
Excellent point, let me rename these to contain `logic`. I hate that Verilog makes time values four-state, but then also tells you that any X or Z bits are treated as 0 for the purpose of delay control or delayed assignments 🤦.

From IEEE 1800-2017 § 6.11 Integer data types:

<img width="652" height="337" alt="Screenshot 2025-08-29 at 09 29 26" src="https://github.com/user-attachments/assets/702fb58e-b431-452d-9e71-02026028dc6e" />